src/data-accessor.ts

```diff
@@ -55,7 +55,15 @@ export const statsForLevel = (level: CharacterLevel, heroClass: HeroClass): Leve
 };
 
 // TODO: This is really silly, this should be in a map
-export const getPowerByName = (name: UniqueCardName) => ALL_POWERS.find((p) => p.name === name);
+export const getPowerByName = (name: UniqueCardName): Power => {
+  const maybePower = ALL_POWERS.find((p) => p.name === name);
+
+  if (maybePower) {
+    return maybePower;
+  }
+
+  throw new Error("Unable to find power data");
```

Comment:
I dont like this because there might be times where you just want to check if a power exists without throwing.

suggestion: make a getPowerByNameOrThrow method that does this
additionally: put the name in the error message so we can more easily debug it if something goes wrong